### ISSUE TITLE
Compatible with Firefox as of version 57?

### ISSUE THREAD

**Issue by jab** [surma/streaming-dot#3]
Just came across this for the first time, saw that https://github.com/surma/streaming-dot#compatibility says Firefox support is still in development due to "Missing [ReadableStream] on fetch (https://bugzilla.mozilla.org/show_bug.cgi?id=1128959)", clicked through to the upstream bug, and saw it's been marked as fixed since the last time this project's README was updated. In case you'd like to update the README. 😊 

Thanks for releasing streaming-dot and for all your other great work!

**Comment by jab** [comment]
According to https://caniuse.com/#feat=textencoder Safari added support for TextDecoder in version 10.1. Another browser that streaming-dot can now declare compatible? 😊

**Comment by surma** [comment]
Yeah you are definitely correct. I have been great at maintaining streaming-dot. I shall bring everything up to speed in 2018!
